tests/unit_tests/test_lr_assignment.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+
+from super_gradients.common.object_names import Models
+from super_gradients.training.models import LeNet
+from super_gradients.training.utils import HpmStruct
+from super_gradients.training.utils.optimizer_utils import separate_lr_groups
+from super_gradients.training import models
+
+
+class TestSeparateLRGroups(unittest.TestCase):
```

Comment:
Kudos for tests 👍 
I think two test-cases are missing:
1) When some layers of models has requires_grad==False. Test should check that total number of optimizable params is correct
2) Test case to check case when `initial_lr = {default: 1, some_layer: 0}`. In this case also number of optimizable params in all params groups should be correct

Reply:
added https://github.com/Deci-AI/super-gradients/commit/966180d62ef7e8c1a5eb1e2a98a2f9a8783c9af3